resources/aws/asg_stack.go

```diff
@@ -19,6 +20,8 @@ const (
 	defaultTimeout = 5
 	// imageIDParam is the Cloud Formation parameter name for the ASG image ID.
 	imageIDParam = "ImageID"
+
+	stackDoesNotExistError = "Stack with id %s does not exist"
```

Comment:
The Cloud Formation API returns a validation error when getting a stack that doesn't exist. So string comparison is needed :(